src/components/ChatRoom/ChatInput.tsx

```diff
@@ -0,0 +1,213 @@
+import { ReactComponent as EmogiIcon } from "asset/icons/EmogiIcon.svg";
+import { ReactComponent as FileInputIcon } from "asset/icons/FileInputIcon.svg";
+import { ReactComponent as AudioIcon } from "asset/icons/AudioIcon.svg";
+import { ReactComponent as SendIcon } from "asset/icons/SendIcon.svg";
```

Comment:
svg 사용하셨네요! 너무 조아요 😆

그런데 이렇게 아이콘이 많이 필요할 때는 코드가 길어질 수 있으니 **index.ts** 생성해서 svg만 모으고 거기서 import 하는 것을 추천드려요 ! 훨씬 코드가 짧아진답니다 >  < 

```suggestion
import { EmogiIcon,  FileInputIcon, AudioIcon, SendIcon } from "../asset";
```